user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+
+=== System Package Managers
+
+Depending on your operating system, you may have different package managers available.
+For instance, on Debian-based Linux distributions, APT is a popular choice, while macOS users often use Homebrew, and Windows users may prefer Chocolatey.
+Depending on your version of Unix, you can use a package manager such as *apt*, *dnf*, *yum*, or *pacman*.
+While using a system package manager has its advantages, there are also some drawbacks to consider.
+These include potential version conflicts with system libraries, lack of control over the installation process, and the risk of using outdated or unsupported versions of the Boost libraries.
+
+==== Linux
+
+Update your package list to ensure you're getting the most recent version of the Boost libraries available in the repositories.
+
+[tabs]
+========
+Ubuntu::
++
+--
+To install the Boost pass:[C++] libraries onto Debian-based systems (such as Debian, Ubuntu), you can use the package manager apt.
+
+[source]
+----
+sudo apt update
+----
+--
+
+Fedora::
++
+--
+To install the Boost pass:[C++] libraries on Fedora, you can use the package manager *dnf*.
+
+[source]
+----
+sudo dnf update
+----
+
+The Boost libraries are usually available as pre-compiled packages in the official Fedora repositories.
+--
+
+CentOS::
++
+--
+To install the Boost pass:[C++] libraries on CentOS, you can use the package manager *yum*.
+
+Note::
+If you are using CentOS 8 or later, you might need to enable the PowerTools repository to get the Boost development libraries:
+
+[source]
+----
+sudo yum config-manager --set-enabled powertools
+----
+
+Then update your package list to ensure you're getting the most recent version of the Boost libraries available in the repositories:
+
+[source]
+----
+sudo yum update
+----
+
+The Boost libraries are usually available as pre-compiled packages in the official CentOS repositories.
+--
+
+Arch::
++
+--
+To install the Boost pass:[C++] libraries onto Arch-based systems (such as Arch Linux, Manjaro), you can use the package manager *pacman*.
+
+[source]
+----
+sudo yum update
+----
+--
+========
+
+Install the Boost development libraries using the package manager.
+
+[tabs]
+========
+Ubuntu::
++
+--
+[source]
+----
+sudo apt install libboost-all-dev
+----
+--
+
+Fedora::
++
+--
+[source]
+----
+sudo dnf install boost-devel
+----
+--
+
+CentOS::
++
+--
+[source]
+----
+sudo yum install boost-devel
+----
+--
+
+Arch::
++
+--
+[source]
+----
+sudo pacman -S boost
+----
+--
+========
+
+Once the installation is complete, the Boost libraries will be installed on your system, typically in `/usr`.
+To verify that the Boost libraries have been installed correctly, you can check the version number:
+
+[source]
+----
+cat /usr/include/boost/version.hpp | grep "BOOST_LIB_VERSION"
+----
+
+This command should output the Boost version installed on your system.
+
+==== Homebrew
```

Comment:
would it make sense to put this into the tabs, too? Looks at the same level as the `apt`/`dnf` commands